### PR TITLE
Ft Help center with FAQs, help videos, live chat 147170309

### PR DESCRIPTION
**Pull request by johnseremba**
### New Features ###
* ### Live Chat ###
   * We integrated a live chatting library called olark
   * You can access the back-end at www.olark.com @dmwiine's account is being used here.
* ### Create FAQ ###
   * Create FAQ Categories (CRUD)
      All FAQ's are grouped according the different categories under-which they fall.
      The System now supports Create, Read, Update and Delete of these Categories. However, these are operations performed by the Super Admin.
   * Create FAQ Item (CRUD)
      FAQ are intuitively created by the super admin. The Answer/body of the FAQ can be nicely styled  with HTML. CKEditor was incorporated to ease this operation.
   * Integrate Django-CKEditor
   * Integrate Django-Fontawesome
   * Integrate Django-Crispy Forms
   * Added Bootstrap modal dialog popups for quick editing and creation of data
* Tests for FAQ (CRUD)
   * We have included tests to test the creation, reading, update and deletion of both FAQ's and their parent Categories.
    These can be found under `hc/front/tests/test_add_faq`
* ### Help Videos ###
   * End users of the system are able to play and view help videos as uploaded by the super administrator. The Super Admin however is solely responsible for uploading of video content.
   * Upload Help Videos to the system
   * Delete Undesired Help Videos
   * Play Help Videos
   * Implemented a function decorator that checks for super admin before running certain sensitive functions.
* Affected Files shall almost all be found under:
   * `hc/front/views/`
   * `hc/front/tests/`
   * `templates/front/`
   * `hc/help_videos/`
   * `templates/help_videos/`

### CODE REVIEW

**Comment by amwaleh** [comment]
Coverage decreased (-77.8%) to 2.372% what happened ??

File: .vscode/settings.json [hunk 1]
```diff
@@ -0,0 +1,3 @@
+// Place your settings in this file to overwrite default and user settings.
```
Comment (amwaleh) [inline .vscode/settings.json]: This file needs to be git ignored

File: hc/help_videos/tests.py [hunk 2]
```diff
@@ -0,0 +1,3 @@
+from django.test import TestCase
```
Comment (amwaleh) [inline hc/help_videos/tests.py:3]: where are the tests???

File: hc/help_videos/urls.py [hunk 3]
```diff
@@ -0,0 +1,15 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+# File: urls.py
+# Author: Collins Abitekaniza <abtcolns@gmail.com>
```
Comment (amwaleh) [inline hc/help_videos/urls.py]: whats the purpose of this?

File: hc/help_videos/views.py [hunk 4]
```diff
@@ -0,0 +1,61 @@
+from django.shortcuts import render
+from django.core.files.storage import FileSystemStorage
+from .models import Video
+from django.http import HttpResponse
+from django.http import HttpResponseForbidden
+from django.contrib.auth.decorators import login_required
+
+# Create your views here.
```
Comment (amwaleh) [inline hc/help_videos/views.py]: delete this comment

File: hc/help_videos/views.py [hunk 5]
```diff
@@ -0,0 +1,61 @@
...
+
+
+def index(request):
+    ctx = {
+        'section': "help-videos",
+        'videos': Video.objects.all()
+    }
+    return render(request, 'help_videos/videos.html', ctx)
+
+# define admin required decorator
```
Comment (amwaleh) [inline hc/help_videos/views.py]: hanging comment... where does this comment belong?

File: hc/help_videos/tests.py (hunk 2 above)
Comment (johnseremba) [reply hc/help_videos/tests.py:3]: Tests were added for the video-upload module

File: .vscode/settings.json (hunk 1 above)
Comment (johnseremba) [reply .vscode/settings.json]: t's been added to .gitignore and removed from the repo

File: hc/help_videos/urls.py (hunk 3 above)
Comment (johnseremba) [reply hc/help_videos/urls.py]: It has been deleted.

File: hc/help_videos/views.py (hunk 5 above)
Comment (johnseremba) [reply hc/help_videos/views.py]: Fixed

File: hc/help_videos/views.py (hunk 4 above)
Comment (johnseremba) [reply hc/help_videos/views.py]: Fixed

**Pull request closed without merging**